src/components/Dashboard/Tiles/Apps/AppTile.tsx

```diff
@@ -26,6 +26,15 @@ export const AppTile = ({ className, app }: AppTileProps) => {
     .join(': ');
 
   const isRow = app.appearance.positionAppName.includes('row');
+  const externalUrl = useMemo(() => {
+    if (app.behaviour.externalUrl.length > 0) {
+      if (app.behaviour.externalUrl.includes('://[homarr_domain]') && window.location.hostname) {
```

Comment:
Doesn't work with subdomains if I read that correctly.

Reply:
Can you give me an example?

Reply:
Example would be homarr.domain.com 

are you able to test that ? @j3lte

Reply:
Well, let's imagine homarr has "https://homarr.domain.com" and "https://homarr.localdomain.com", and sonarr is on "https://sonarr.domain.com" and "https://sonarr.localdomain.com", each being there depending if you're home or external.
You would then do "https://sonarr.[homarr_domain]" in the config.
They share domain but not the subdomain, and the subdomain is separating the domain from the "://", so testing for it means subdomains won't work. At least that's how I interpret it.

Reply:
Fair enough, unfortunately `subdomain` is not part of `window.location`. So in the case of "homar.domain.com", `window.location.host` and `window.location.hostname` are both "homar.domain.com", not "domain.com". Your proposal with both protocol and domain separated would work if homarr is at "domain.com" and sonar at "sonar.domain.com"

Reply:
[homarr_hostname] -> returns homarr's full base url including it's current subdomain
[homarr_domain] -> returns homarr's domain with subdomain filtered out
[homarr_protocol] -> return homarr's protocol (http/s)
[homarr_baseUrl] -> full address excluding port and path

You could manually filter out subdomains, but I agree it could be a bit of a pain.
I found this package that does it for you, and quite well at that: https://www.npmjs.com/package/tldts
It adds a new dependency but frankly we could use it to better format our URL's in integrations so it's fine.

I'm sorry if it may feel like I'm nagging for contributions, I just feel like your idea has that much potential and should be implemented to it's fullest.